verilog/analysis/verilog_linter.cc

```diff
@@ -210,6 +211,26 @@ LinterConfiguration LinterConfigurationFromFlags() {
   const auto& ruleset = absl::GetFlag(FLAGS_ruleset);
   config.UseRuleSet(ruleset);
 
+  // Read local configuration file
+  std::ifstream local_config(".rules.verible_lint");
```

Comment:
We should probably define a flag `--rule_config` (paging @fangism if he has a particular preference) and use that instead of the hardcoded `.rules.verible_lint` (which we still can make the default).

Reply:
Sounds ok.

Reply:
Sounds good.

Reply:
Added a flag, thanks!